package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "precommit": "lint-staged",
-    "prepare": "lerna bootstrap",
```

Comment:
I think we can ditch the `bootstrap` npm run script if this does the same thing, e.g. `"prepare": "lerna bootstrap --hoist"`

Reply:
Ditched `bootstrap`, and use `lerna bootstrap --hoist` as the source of `prepare`.